hack/update-codegen.sh

```
@@ -8,7 +8,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-CODE_GEN_DIR=$(go list -m -f '{{.Dir}}' k8s.io/code-generator)
+CODE_GEN_DIR=$(GOWORK=off go list -m -modfile $(go list -m -f '{{.Dir}}' github.com/gardener/gardener/hack/tools)/go.mod -f '{{.Dir}}' k8s.io/code-generator)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)